binaries/setup-cloud-scoot/main.go

```diff
@@ -0,0 +1,22 @@
+package main
```

Comment:
I think this just connects you to a cloud scoot, which if we had an open source cloud scoot that ran in the cloud, this would also find the address (using service discovery) to connect you to that, so the fact that it's local is just kinda temporary.